ecl/hthor/hthor.cpp

```diff
@@ -8213,8 +8213,7 @@ void CHThorDiskReadBaseActivity::resolve()
         {
             StringBuffer buff;
             buff.appendf("Input file '%s' was missing but declared optional", mangledHelperFileName.str());
-            UWARNLOG("%s", buff.str());
```

Comment:
Were we logging twice before ?  ie once with this UWARNLOG() and then also in addWuException() ?

Reply:
That's correct.  It was writing twice before. Quadruple checked to make sure.  Once here and once in EclAgent::addWuException ->  agent.addWuException() -> ecl/eclagent/eclagent.cpp's global function addException()